source/includes/fundamentals/code-examples/crud/change/Replace.cs

```diff
@@ -0,0 +1,79 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+
+namespace CSharpExamples.Fundamentals.ReplaceOne;
+
+public class ReplaceOne
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private const string MongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Create filter 
+        var filter = Builders<Restaurant>.Filter.Eq(restaurant => restaurant.Name, "Pizza Town");
+
+        // Find restaurant named "Pizza Town"
+        var oldRestaurant = _restaurantsCollection.Find(filter).First();
+        Console.WriteLine($"Restaurant with ID {oldRestaurant.Id} before replacement: {oldRestaurant.Name}");
+
+        // Replace one document synchronously
+        var syncResult = ReplaceOneRestaurant();
+        Console.WriteLine($"Restaurants modified by replacement: {syncResult.ModifiedCount}");
+
+        var firstPizzaRestaurant = _restaurantsCollection.Find(filter).First();
+        Console.WriteLine($"Restaurant with id {oldRestaurant.Id} after replacement: {firstPizzaRestaurant.Name}");
+
+        Write("Resetting sample data...");
```

Comment:
This probably does not compile. Maybe it's best to double check that all recent changes compile.